uber/templates/regform.html

```diff
@@ -415,10 +422,19 @@
 <div class="form-group">
     <label for="email" class="col-sm-3 control-label">Email Address</label>
     <div class="col-sm-6">
-        <input type="email" type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address">
+        <input type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address">
```

Comment:
This `type` field is meant to do some (very basic) HTML5 email validation. Did it cause a problem?